Frameworks/iOS/fat/MUXSDKStats.framework/Headers/MUXSDKStats.h

```diff
@@ -49,6 +49,19 @@ FOUNDATION_EXPORT
  */
 + (MUXSDKPlayerBinding *_Nullable)monitorAVPlayerViewController:(nonnull AVPlayerViewController *)player withPlayerName:(nonnull NSString *)name playerData:(nonnull MUXSDKCustomerPlayerData *)playerData videoData:(nullable MUXSDKCustomerVideoData *)videoData;
 
+/*!
+ @method      monitorAVPlayerViewController:withPlayerName:playerData:videoData:withAutomaticErrorTracking:
+ @abstract    Starts to monitor a given AVPlayerViewController.
+ @param       player An AVPlayerViewController to monitor
+ @param       name A name for this instance of the player
+ @param       playerData A MUXSDKCustomerPlayerData object with player metadata
+ @param       videoData A MUXSDKCustomerVideoData object with video metadata
+ @param       automaticErrorTracking boolean to indicate if the SDK should automatically track player errors
+ @return      an instance of MUXSDKAVPlayerLayerBinding or null
+ @discussion  Use this method to start a Mux player monitor on the given AVPlayerViewController. The player must have a name which is globally unique. The config provided should match the specifications in the Mux docs at https://docs.mux.com
+ */
++ (MUXSDKPlayerBinding *_Nullable)monitorAVPlayerViewController:(nonnull AVPlayerViewController *)player withPlayerName:(nonnull NSString *)name playerData:(nonnull MUXSDKCustomerPlayerData *)playerData videoData:(nullable MUXSDKCustomerVideoData *)videoData withAutomaticErrorTracking:(BOOL)automaticErrorTracking;
```

Comment:
Small nit about the naming here since we already have the `with` in `withPlayerName` I'd go with 
`monitorAVPlayerViewController: withPlayerName: playerData:videoData:automaticErrorTracking`

Reply:
yeah that's cleaner 👍